src/super_gradients/recipes/dataset_params/imagenet_mobilenetv3_dataset_params.yaml

```diff
@@ -0,0 +1,15 @@
+defaults:
+  - imagenet_dataset_params
+
+batch_size: 128
+
+# TODO: REMOVE ABOVE, HERE FOR COMPATIBILITY UNTIL WE REMOVE DATASET_INTERFACE
+#defaults:
```

Comment:
Redundant commented out code.

Reply:
When we will remove the section above we need to keep the defaults. In other sections we have other params that we also want to keep (like img_mean).
To make sure that we don't forget these params when we will remove the section above I already wrote and commented the lines of code that we will still need.
I added a small comment to explain that this code should be uncommented it when the above part is removed: dd7a66c25443ec5d09d590c2c92cc6750e0f5cd7